ZitiUpdateService/UpdateService.cs

```diff
@@ -293,14 +293,18 @@ public partial class UpdateService : ServiceBase {
 		}
 
 		private void outputExternalIP(string destinationFolder) {
-			Logger.Info("capturing external IP address using eth0.me");
+			Logger.Info("capturing external IP address using dig command");
 			try {
+				Process process = new Process();
+				ProcessStartInfo startInfo = new ProcessStartInfo();
+				startInfo.WindowStyle = ProcessWindowStyle.Hidden;
+				startInfo.FileName = "cmd.exe";
 				var extIpFile = Path.Combine(destinationFolder, "externalIP.txt");
-				System.Net.Http.HttpClient httpClient = new System.Net.Http.HttpClient();
-				var resp = httpClient.GetAsync("http://eth0.me").Result;
-				resp.EnsureSuccessStatusCode();
-				string responseBody = resp.Content.ReadAsStringAsync().Result;
-				File.WriteAllText(extIpFile, responseBody);
+				Logger.Debug("running dig +short myip.opendns.com @resolver1.opendns.com to {0}", extIpFile);
+				startInfo.Arguments = $"/C dig +short myip.opendns.com @resolver1.opendns.com > \"{extIpFile}\"";

```

Comment:
dig is not a portable windows command. you cannot use this:
```
C:\Users\WDAGUtilityAccount>dig
'dig' is not recognized as an internal or external command,
operable program or batch file.
```

why are you changing this behavior?

Reply:
eth0.me is not accessible by some of our customers, because that external url is blocked. So we may need to find any other option to get the external ip

Reply:
I changed it to nslookup